src/shared_utilities/Domains.jl

```diff
@@ -267,22 +267,27 @@ function Plane(;
         @assert periodic == (false, false)
         radius_earth = FT(radius_earth)
         long, lat = longlat
+        dxlim = xlim # long
+        dylim = ylim # lat
```

Comment:
```suggestion
        dxlim = xlim # lat
        dylim = ylim # long
```
I had to switch things around to work with climacore

Reply:
I thought though that the user passes in xlim refering to long, and ylim refering to lat, and "longlat". And then here internally we swap them to work with climacore. I updated the comments to how I understand it, but let me know if the user actually passes in xlim refering to lat, ylim refering long.

Reply:
Ah yes. You were and are right!